### ISSUE TITLE
conflict with base64 versioning

### ISSUE THREAD

**Issue by uidzip** [mogest/prawn-svg#161]
Sinatra 4.0.0 requires base64 version 0.2.0.  prawn-svg currently has "~> 0.1.0".  This makes running a Sinatra app that uses prawn-svg impossible with Sinatra 4.0.0.

Backing down to Sinatra 3.2 is a lot of work, and ultimately, didn't work for me.  There was something else I haven't identified yet that also requires base64 0.2.0, I just don't know what it is yet.

```
toml@test:~/focusforge/system/proc$ puma -C httpapi_puma.rb
Puma starting in single mode...
* Puma version: 6.4.2 (ruby 3.2.3-p157) ("The Eagle of Durango")
*  Min threads: 0
*  Max threads: 5
*  Environment: development
*          PID: 2540
**Unable to activate prawn-svg-0.34.0, because base64-0.2.0 conflicts with base64 (~> 0.1.0)**
```

```
irb(main):001:0> require 'sinatra'
=> true
irb(main):002:0> require 'prawn-svg'
/usr/local/lib/ruby/site_ruby/3.2.0/rubygems/specification.rb:2317:in `raise_if_conflicts': Unable to activate prawn-svg-0.34.0, because base64-0.2.0 conflicts with base64 (~> 0.1.0) (Gem::ConflictError)  
```

**Comment by mogest** [comment]
We've just had a PR merged removing the base64 dependency—I'll be releasing this sometime in the next week once a few more changes have landed.

**Comment by mojavelinux** [comment]
I hate to be that person, but would it be possible to release a version of prawn-svg with this fix? It's now causing warnings in Ruby 3.3 when using prawn-svg as a transitive dependency.

**Comment by mojavelinux** [comment]
Although the base64 gem was added to the gemspec in the 0.34.0, subsequent unreleased commits removed the dependency altogether. 0.34.0 also had other issues which have since been fixed. So what we're looking for is a 0.35.0 release.

**Comment by mogest** [comment]
All good, I've just released 0.34.1.

**Comment by mojavelinux** [comment]
Thanks a bunch!!
